docs/getting-started/app-custom-theme.md

```diff
@@ -273,6 +273,71 @@ const LogoFull = () => {
 };
 ```
 
+## Custom Icons
+
+You can also customize the Project's _default_ icons.
+
+You can change the following [icons](https://github.com/backstage/backstage/blob/master/packages/app-defaults/src/defaults/icons.tsx).
+
+### Requirements
+
+- Files in `.svg` format
+- React components created for the icons
+
+### Create React Component
+
+In your front-end application, locate the `src` folder. We suggest creating the `assets/icons` directory and `CustomIcons.tsx` file.
+
+> Another example [here](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/src/components/AzurePipelinesIcon/AzurePipelinesIcon.tsx), if you want to ensure proper behavior in light and dark themes.
+
+```tsx
+// customIcons.tsx
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+
+import React from 'react';
+
+export const HeaderIcon = (props: SvgIconProps) => (
```

Comment:
```suggestion
export const ExampleIcon = (props: SvgIconProps) => (
```

Reply:
This should match the example below where we import it so it should probably be ExampleIcon here :)

Reply:
Wow, I forgot to update this "guy" haha
Thanks for the review!